### PR TITLE
Better reporting of CPU timing

### PR DESCRIPTION
**Pull request by Dr15Jones**
#### PR description:

- The Timing service reported CPU time for child processes as part of the total CPU time, but not as part of the loop time.
- The timing summary did not account for child processes at all.

#### PR validation:

Code compiles.

### CODE REVIEW

**Comment by Dr15Jones** [comment]
please test

File: FWCore/Utilities/src/ChildrenCPUTimer.cc [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+// constructors and destructor
+//
+ChildrenCPUTimer::ChildrenCPUTimer() : state_(kStopped), startCPUTime_(), accumulatedCPUTime_(0) {
+  startCPUTime_.tv_sec = 0;
+  startCPUTime_.tv_usec = 0;
+}
+
+// ChildrenCPUTimer::ChildrenCPUTimer(ChildrenCPUTimer const& rhs) {
+//    // do actual copying here;
+// }
```
Comment (makortel) [inline FWCore/Utilities/src/ChildrenCPUTimer.cc:40]: Is this commented code needed?

File: FWCore/Utilities/src/ChildrenCPUTimer.cc [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+//
+// assignment operators
+//
+// ChildrenCPUTimer const& ChildrenCPUTimer::operator=(ChildrenCPUTimer const& rhs) {
+//   //An exception safe implementation is
+//   ChildrenCPUTimer temp(rhs);
+//   swap(rhs);
+//
+//   return *this;
+// }
```
Comment (makortel) [inline FWCore/Utilities/src/ChildrenCPUTimer.cc:53]: Is this commented code needed?

**Comment by Dr15Jones** [comment]
Please test

**Comment by makortel** [comment]
+1

**Comment by perrotta** [comment]
+1

**Pull request merged**